definitions/npm/redux-observable_v0.17.x/test_redux-observable_v0.17.x.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+import type { Epic, EpicMiddleware } from 'redux-observable';
+import { ActionsObservable, combineEpics, createEpicMiddleware } from 'redux-observable';
+
+// Redux misc
+type State = Array<number>;
+type Action = { type: 'A' | 'B' };
+
+/*
+ * Epic type
+ */
+const epic1: Epic<State, Action, void> =
+  (action$) =>
+    action$.filter(action => action.type === 'A')
+      .mapTo({ type: 'B' });
+const epic2: Epic<State, Action, void | number> =
```

Comment:
I don't think this is valid... You might need to pick between `void` and `number`.